src/Container.Database.MySql/MySqlContainer.cs

```diff
@@ -39,8 +40,46 @@ private static IImage CreateDefaultImage(IDockerClient dockerClient, ILoggerFact
 
         private string _connectionString;
 
+        /*
+         * NOTE: MySqlConnector made a breaking change in v1.0 by changing the namespace.
+         * We have to try loading multiple types of DbProviderFactory depending on the installed version.
+         * Once found, the DbProviderFactory is cached in this Lazy instance.
+         */
+        private static readonly Lazy<DbProviderFactory> s_lazyDbProviderFactory = new Lazy<DbProviderFactory>(() =>
+        {
+            var providerFactoryTypes = new[]
```

Comment:
Could I get you to refactor this method into a class on its own? Say `MultiVersionMySqlConnectorFactory` or something.

So that `providerFactoryTypes` can now be a constant.

Reply:
I made the requested change. Tell me if it's okay for you :)

Note: I have suggestions coming from `.editorconfig` saying that I should prefix the static members with `s_`. But it seems this format is never used, so I ignored it.